pkg/limayaml/defaults_test.go

```diff
@@ -32,6 +32,8 @@ func TestFillDefault(t *testing.T) {
 		arch = X8664
 	case "arm64":
 		arch = AARCH64
+	case "arm":
+		arch = ARMV7L // assume runtime.GOARM == 7
```

Comment:
Can we panic if runtime.GOARM != 7 ?

Reply:
The thing is that there _is_ no runtime.GOARM, there is only a $GOARM. So we would have to reimplement:

> goarm 6 requires: cpu.ARM.HasVFP
> goarm 7 requires: cpu.ARM.HasVFPv3

But this is also a host check (I think), so it seems unlikely that it would even be tried on `armv5` or `armv6`?

This whole arm32 target is likely to be emulation-only, I think Raspberry Pi 3* was the first one with KVM...

\* i.e. GOARCH=arm64 and `armv8`

The use case was for testing programs intended for embedded devices, that required a 32-bit kernel too.

Otherwise you would probably just run a `--platform=linux/arm` container, with qemu-user, instead.